static/js/pages/breakdowns.js

```diff
@@ -0,0 +1,204 @@
+'use strict';
+
+/* global context, ga */
+
+var $ = require('jquery');
+var _ = require('underscore');
+var analytics = require('fec-style/js/analytics');
+var helpers = require('../modules/helpers');
+var ReactionBox = require('../modules/reaction-box').ReactionBox;
+var moment = require('moment');
+
+var TOP_ROW = _.template(
+  '<div role="row" class="simple-table__row">' +
+    '<div class="simple-table__cell"><a href="{{ url }}">{{ name }}</a></div>' +
+    '<div class="simple-table__cell t-right-aligned">{{ amount }}</div>' +
+    '<div class="simple-table__cell"><div class="bar-container">' +
+      '<div class="value-bar" data-value="{{ value }}" data-party="{{ party }}"></div>' +
+    '</div></div>',
+    {interpolate: /\{\{(.+?)\}\}/g}
+);
+
+// Store candidate office letters for to look up when a chart category is a candidate
+var candidateCategories = ['P', 'S', 'H'];
+
+function TopEntities(elm, type) {
+  this.$elm = $(elm);
+  this.type = type === 'raising' ? 'receipts' : 'disbursements';
```

Comment:
Ah, found a simpler way to do this by just passing `receipts` or `disbursements` directly in as the param instead of deriving it.